packages/animation-curve-block/src/components/AnimationCurveFlyout.tsx

```diff
@@ -93,10 +93,9 @@ export const AnimationCurveFlyout = ({
 
     return (
         <Flyout
-            trigger={<></>}
+            trigger={(props, ref) => <div ref={ref as MutableRefObject<HTMLDivElement>} />}
```

Comment:
What is the purpose of this change?

Reply:
Its to get rid of the trigger button which would be focusable but as its open via the toolbar this would be quite confusing for the user.

Or described by Martin:
`Accessibility: Edit animation button seems to be targeted twice when going through with tab. Second time there is no visual highlight, which could be confusing.`